### PR TITLE
joule++

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by cfouant** [comment]
NACK by @komidore64

**Comment by komidore64** [comment]
@cfouant, get three folks from `#rdu16ne` to ack, and i'll merge it.

**Comment by asmacdo** [comment]
ACK

File: lib/ircle/plugins/lunchbot/restaurants.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - "gravy"
 - "gringo a go-go"
 - "jimmy v's"
+- "joule"
 - "oxford"
```
Comment (asmacdo) [inline lib/ircle/plugins/lunchbot/restaurants.yml]: oxford is on here???? really?

**Comment by jeremycline** [comment]
NACK

**Comment by cfouant** [comment]
@jeremycline--

**Comment by cfouant** [comment]
@jeremycline--

**Comment by komidore64** [comment]
ACK, because it was brought to my attention that their menu has changed since we last were there. I'm willing to try it again.

**Comment by cfouant** [comment]
YUSSSS

File: lib/ircle/plugins/lunchbot/restaurants.yml (hunk 1 above)
Comment (jlsherrill) [inline lib/ircle/plugins/lunchbot/restaurants.yml]: lol oxford gets the hate but not "el rodeo" ?

**Comment by jlsherrill** [comment]
ACCCCKKKKKKK

**Comment by cfouant** [comment]
ACK!

**Comment by komidore64** [comment]
3+ ACKs from others. merging.

**Comment by jeremycline** [comment]
Don't go breaking my heart, @komidore64

**Comment by komidore64** [comment]
@jeremycline, the tribe has spoken.

**Comment by jeremycline** [comment]
(╯°□°）╯︵ ┻━┻

**Pull request merged**